pkg/controller/build/buildrequest/machineosbuild_test.go

```diff
@@ -81,6 +80,7 @@ func TestMachineOSBuild(t *testing.T) {
 				OSImageURLConfig:  fixtures.OSImageURLConfig(),
 			},
 		},
+		/* QOCL: Are these test cases valid anymore?
```

Comment:
There are 6 test cases here that seem to exercise fields that have been removed: `BaseOSExtensionsImagePullspe`, `BaseOSImagePullspec` and `ReleaseVersion`

Reply:
In my opinion, we still need those fields in order to produce a hashed name for a MachineOSBuild. The difference is that we only have a single canonical source for these values with the new API changes.